charts/pega/values-large.yaml

```diff
@@ -274,9 +274,10 @@ installer:
   # Upgrade specific properties
   upgrade:
     # Type of upgrade
-    # Valid values are 'in-place' , 'out-of-place'
+    # Valid values are 'in-place' , 'zero-downtime' , 'custom' , 'out-of-place-rules' , 'out-of-place-data';
     upgradeType: "in-place"
     # Specify target rules schema for migration and upgrade
     targetRulesSchema: ""
     # Specify target data schema for migration and upgrade.
+    #Support for postgres db - 8.4.0 and later , oracle db - 8.4.3 and later
```

Comment:
This needs some deep explanation...not sure if it's clear...
# For postgres databases that you are upgrading from Pega Infinity version 8.4.0 and later
# or for Oracle databases that you are upgrading from Pega Infinity version 8.4.3 and later,
# Specify a name for a target data schema that the upgrade process creates for patches and upgrades.

Reply:
yes, we need space after `#` in comment. yaml lint validation in travis build would fail otherwise.